src/main/java/org/team199/wpiws/connection/WSConnection.java

```diff
@@ -94,11 +102,35 @@ public void onMessage(String message) {
         @Override
         public void onClose(int code, String reason, boolean remote) {
             ConnectionProcessor.onClose(this, code, reason, remote);
+            if(autoReconnect && remote) {
+                System.out.println("Reconnnecting...");
+                RunningObject.start(this::tryReconnect);
+            }
         }
 
         @Override
         public void onError(Exception e) {
-            ConnectionProcessor.onError(this, e);
+            if(e instanceof ConnectException) {
+                //Connection Refused
+                System.err.println("Error: Connection Refused! Trying again in " + retryTimeout + " second(s)");
+                int waitSecs = retryTimeout++;
```

Comment:
Couldn't this result in an indefinitely long wait between retries? Recommend capping it at say 1 sec.